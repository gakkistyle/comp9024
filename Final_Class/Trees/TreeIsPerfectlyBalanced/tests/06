1 2 4 7 5 3 6
7 4 2 5 1 6 3
