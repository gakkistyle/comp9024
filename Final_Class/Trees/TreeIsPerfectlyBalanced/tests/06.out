Enter the preorder traversal of the tree: Enter the in-order traversal of the tree: Tree:

        1
       / \
      /   \
     /     \
    2       3
   / \     /
  4   5   6
 /
7

TreeIsPerfectlyBalanced returned: FALSE
